internal/proto/data_coord.proto

```diff
@@ -397,6 +398,15 @@ message WatchChannelsResponse {
   common.Status status = 1;
 }
 
+message SetSegmentStateRequest {
+  int64 segment_id = 1;
```

Comment:
please add `common.MsgBase` as first member for a request message.

Reply:
DONE.